src/renderer/components/ParticipantsTile/index.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <section class="tile is-parent">
+    <div class="tile is-child box">
+      <h3 class="title">Meeting participants</h3>
+
+      <participant-list v-bind:list="list"></participant-list>
+      <add-participant :onAdded="onParticipantAdded"></add-participant>
```

Comment:
Here you should listen to event I mentioned earlier: `@on-added="onParticipantAdded"`